client/src/components/DeckBrowser.js

```diff
@@ -75,36 +74,7 @@ const DeckBrowser = () => {
           </Container>
         </div>
         <Container className={classes.cardGrid} maxWidth="md">
-          <Grid container spacing={4}>
-            {cards.map((card) => (
-              <Grid item key={card} xs={12} sm={6} md={4}>
-                <Card className={classes.card}>
-                  <CardContent className={classes.cardContent}>
-                    <Typography gutterBottom variant="h5" component="h2">
-                      Deck name
-                    </Typography>
-                    <Typography gutterBottom>
-                      Score:
-                    </Typography>
-                    <Typography variant="caption">
-                      Created on [date]
-                    </Typography>
-                  </CardContent>
-                  <CardActions>
-                    <Button size="small" color="primary">
-                      Practice
-                    </Button>
-                    <Button size="small" color="primary">
-                      Quiz
-                    </Button>
-                    <Button size="small" color="secondary">
-                      Edit
-                    </Button>
-                  </CardActions>
-                </Card>
-              </Grid>
-            ))}
-          </Grid>
+          <DeckCollection />
```

Comment:
I was unable to comment on line `45` since there was no diff, but it looks like `const cards = [1, 2, 3, 4, 5, 6, 7, 8, 9];` is a leftover variable for placeholder cards before you set up the `<DeckCollection />` component!

Reply:
oh ha, whoops! good call.